app/locales/en.js

```diff
@@ -995,7 +995,7 @@ export default {
           workflowTitle: '* **Workflow title:** Give your workflow a short, but descriptive name. If you have multiple workflows and give volunteers the option of choosing which they want to work on, this Workflow title will appear on a button instead of "Get started!"',
           version: '* **Version:** Version indicates which version of the workflow you are on. Every time you save changes to a workflow, you create a new version. Big changes, like adding or deleting questions, will change the version by a whole number: 1.0 to 2.0, etc. Smaller changes, like modifying the help text, will change the version by a decimal, e.g. 2.0 to 2.1. The version is tracked with each classification in case you need it when analyzing your data.',
           tasks: '* **Tasks:** There are two main types of tasks: questions and drawing. For question tasks, the volunteer chooses from a list of answers but does not mark or draw on the image. In drawing tasks, the volunteer marks or draws directly on the image using tools that you specify. They can also give sub-classifications for each mark. Note that you can set the first task from the drop-down menu.',
-          mainText: '* **Main Text:** Describe the task, or ask the question, in a way that is clear to a non-expert. The wording here is very important, because you will in general get what you ask for. Solicit opinions from team members and testers before you make the project public: it often takes a few tries to reach the combination of simplicity and clarity that will guide your volunteers to give you the inputs you need. You can use markdown in the main text.',
+          mainText: '* **Main Text:** Describe the task, or ask the question, in a way that is clear to a non-expert. Markdown can be used only to add images (with alt text), bold and italic text.',
```

Comment:
As far as I can see, this translation key is not actually used anywhere in PFE.

Reply:
I think it may be more than that - the _whole `workflowDetails` key_ (as in `lab.help.howToBuildProject.workflowDetails.mainText`) appears to be no longer used.

As far as I can tell, there was a page called `how-to-build-project.jsx` [(last edit: 2018?)](https://github.com/zooniverse/Panoptes-Front-End/blob/940edc5a5c5a979e93bb271bf710ca4002649787/app/pages/lab/help/how-to-build-project.jsx) that's pretty much gone now.

**UPDATE** OK, found it.
- The `app/pages/lab/help` folder was removed in #5061 
- A lot of the help text has been moved to https://help.zooniverse.org/getting-started/ (aka https://github.com/zooniverse/help )